packages/ovm-toolchain/test/config/truffle-config.js

```diff
@@ -21,7 +21,13 @@ module.exports = {
       network_id: 108,
       networkCheckTimeout: 100000,
       provider: function() {
-        return ProviderWrapper.wrapProviderAndStartLocalNode(new HDWalletProvider(mnemonic, "http://127.0.0.1:8545/", 0, 10));
+        return ganache.provider({
+          mnemonic: mnemonic,
+          network_id: 108,
+          default_balance_ether: 100,
+          gasLimit: 10000000,
+          gasPrice: 0,
```

Comment:
Should this use the values set above?
```suggestion
          gasLimit: gas,
          gasPrice,
```

Reply:
It doesn't seem to have arisen from this PR, but with the account abstraction stuff and pay-for-gas-on-L1, I think `gasPrice` is basically meaningless in the OVM context, unless we end up using a default account wallet which uses that field.